internal/fs/fs_local_vss.go

```diff
@@ -48,7 +50,7 @@ func (fs *LocalVss) DeleteSnapshots() {
 
 	for volumeName, snapshot := range fs.snapshots {
 		if err := snapshot.Delete(); err != nil {
-			fs.msgError("failed to delete VSS snapshot: %s", err)
+			fs.msgError("failed to delete VSS snapshot", err)
```

Comment:
"failed to delete ..." should be part of the error. `item` should be the affected path name, that is `volumeName` in this case. The `item` parameter of `p.Error` is only used by the JSON output but not by the text format.

Reply:
Fixed in https://github.com/restic/restic/pull/2274/commits/2490d13c14fcab71ececa8a1182bde3043f94205